prometheus_client/core.py

```diff
@@ -185,9 +185,8 @@ def __init__(self, name, documentation, value=None, labels=None):
         Metric.__init__(self, name, documentation, 'untyped')
         if labels is not None and value is not None:
             raise ValueError('Can only specify at most one of value and labels.')
-        if labels is None:
-            labels = []
-        self._labelnames = tuple(labels)
+
+        self._labelnames = tuple(labels or [])
```

Comment:
What performance difference does this make?

Reply:
No difference, just more shorted and more Pythonyc.
If you prefer, I can remove this changes from PR.

Reply:
It's fine as-is, and I'd expect it to be faster this way

Reply:
Updated.